troubleshooting/couch2pgOom

```diff
@@ -0,0 +1,38 @@
+# Fixing Couch2pg Out of Memory errors
```

Comment:
Looks good.

Please add a step to back up the document for further inspection.